docs/onduty/onduty.md

```diff
@@ -0,0 +1,54 @@
+# Community On-Duty
+
+Community on-duty is a weekly rotation of Crossplane community members who have
+volunteered to help the Crossplane community. Volunteers are on-duty for a week
+at a time. The goals of the on-duty rotation are to:
+
+1. Ensure the Crossplane community feels heard and supported.
+1. Prevent issues and pull requests (PRs) from falling through the cracks.
+1. Make it easier for the next person on-duty to achieve the above goals.
+
+Volunteers should be able to commit to focus on being on-duty for the majority
+of their typical working day (e.g. ~7 hours a day). On-duty is expected to
+respond to questions, issues and pull requests at their earliest convenience,
+within 24 hours.
+
+> Don't hesitate to reach out to the current person on-duty if you're interested
+> in volunteering but can't commit to so much time - we may be able to work out
+> smaller shifts.
+
+When on-duty, you should:
+
+* Make sure the title of #general in Crossplane Slack indicates that you are the
+  person folks should ping with questions.
```

Comment:
We may want to create a shared Google calendar to keep track of this.

Reply:
Not sure if it requires paid slack, but I have seen folks have an "on-call" team that you can @ mention and it will notify the appropriate person / people.

Reply:
looks like "user groups" require a paid plan in Slack @hasheddan: https://slack.com/help/articles/212906697-Create-a-user-group

> Standard, Plus, and Enterprise Grid plans